src/test/resources/jmxint_container.yaml

```diff
@@ -0,0 +1,13 @@
+init_config:
+  is_jmx: true
+  collect_default_metrics: false
+  collect_default_jvm_metrics: false
+
+instances:
+    -   name: jmxint_container
+        host: {0}
+        collect_default_jvm_metrics: false
+        port: {1,number,#}
+        conf:
+          - include:
+              domain: Bohnanza
```

Comment:
would be nice to have this templatized as well, ties into the "need more control over beans" per test

Reply:
This will go away entirely in favor of literals embedded in each test